src/main/kotlin/no/nav/syfo/StedtilknytningResponseDTO.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.syfo
+
+import no.nav.syfo.behandlendeenhet.BehandlendeEnhet
+import no.nav.syfo.domain.Stedtilknytning
+
+data class StedtilknytningResponseDTO(

```

Comment:
Hva med å kalle det `EnhetstilknytningResponseDTO`? Det er vel alltid enheter vi knytter dem til? 😊
EDIT: Eller kalle det for `BehandlendeEnhet`? Synes ikke `BehandlendeEnhet` navnet på typen lenger gir mening når både `geografiskEnhet` og `oppfolgingsenhet` har typen. Begge er jo uansett ikke "behandlende"
```suggestion
data class EnhetstilknytningResponseDTO(

```

Reply:
Jeg synes `Enhetstilknytning` høres bra ut og virker mer beskrivende enn `BehandlendeEnhet`.